modules/tinymce/src/core/main/ts/annotate/Wrapping.ts

```diff
@@ -28,7 +28,7 @@ export type Decorator = (
 };
 
 const applyWordGrab = (editor: Editor, rng: Range): void => {
-  const r = ExpandRange.expandRng(editor, rng, [{ inline: true }]);
+  const r = ExpandRange.expandRng(editor, rng, [{ inline: 'span' }]);
```

Comment:
I believe that's the case, as previously it used `true` which would never match. However, I can't be sure and a span is what's always used for annotations (see a few lines above) so I do want this to always explicitly be a span. Additionally, adding that type of comment could become outdated/incorrect easily, so I really want to avoid adding that.